routes/movies.routes.js

```diff
@@ -0,0 +1,69 @@
+const Celebrity = require("../models/Celebrity.model");
+const { populate } = require("../models/Celebrity.model");
```

Comment:
What is the purpose of this line? 

`populate()` is a method we can call on any model. We can just call it if we have mongoose dependency